### PR TITLE
Add a simple stubbed-out tile layer that can be extended for layers t…

### PR DESCRIPTION
**Pull request by unbrenzys**
…hat do not require data, or have static data.

### CODE REVIEW

File: scripts/layer/core/Stub.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const lumo = require('lumo');
+
+/**
+ * Simple stub layer that allows the creation of a lumo.TileLayer with no data, or data provided by an extended class
```
Comment (zeroviscosity) [inline scripts/layer/core/Stub.js]: Very minor, but for consistency there should be a period at the end of this sentence.

File: scripts/layer/core/Stub.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	 * Instantiates a stub layer.
+	 *
+	 * @param {Object} options - The options parameter.
+	 */
+	constructor(options) {
+		super(options);
+	}
+
+	/**
+	 * Stub for requesting tile data.  Allows extended class to provide it's own data (if applicable).
```
Comment (zeroviscosity) [inline scripts/layer/core/Stub.js]: One more minor thing: `its` not `it's`

File: scripts/layer/core/Stub.js (hunk 1 above)
Comment (unbrenzys) [reply scripts/layer/core/Stub.js]: Done (and squashed).

File: scripts/layer/core/Stub.js (hunk 2 above)
Comment (unbrenzys) [reply scripts/layer/core/Stub.js]: Done

**Pull request merged**